civihr_employee_portal/views/civihr_employee_portal.views.inc

```diff
@@ -1025,6 +1025,50 @@ function civihr_employee_portal_views_pre_render(&$view) {
   if ($view->name == "approvals") {
     $view->result = removeAbsencesForOtherManagers($view->result);
   }
+  
+  if ($view->name == "civihr_report_people" && $view->current_display == 'civihr_report_table_people' ) {
+    formatLengthsOfService($view->result);
+  }
+}
+
+/**
+ * Processes results in people report view to format length of service for each
+ * contact in a human readable form.
+ * 
+ * @param array $results
+ *   Array of objects (stdClass) that hold data for each contact in people report.
+ */
+function formatLengthsOfService($results) {
```

Comment:
Yes we need this because it being used by Data table to show Employee length of service , so the frontend function is used to show & calculate on Pivot table.